docs/sources/reference/components/otelcol/otelcol.exporter.splunkhec.md

```diff
@@ -0,0 +1,209 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/otelcol/otelcol.exporter.splunkhec/
+description: Learn about otelcol.exporter.splunkhec
+aliases:
+  - ../otelcol.exporter.splunkhec/ # /docs/alloy/latest/reference/components/otelcol.exporter.splunkhec/
```

Comment:
```suggestion
description: Learn about otelcol.exporter.splunkhec
```

New topics don't need an alias. We only need to add these if we move or rename an existing/published topic.